### PR TITLE
HOURS-128 Reenable wifi density and implement a management UI

### PR DESCRIPTION
**Pull request by barmintor**

### CODE REVIEW

**Comment by barmintor** [comment]
This branch is deployed on dev

File: .ruby-version [hunk 1]
```diff
@@ -1 +1 @@
-ruby-2.7.5
\ No newline at end of file
+ruby-3.0.3
```
Comment (elohanlon) [inline .ruby-version:1]: 🎉

File: app/views/locations/show.html.erb [hunk 2]
```diff
@@ -4,6 +4,9 @@
     <div class="col-md-12">
       <h2 class="location-name"><%= link_to @location.name, @location.url %></h2>
       <div class="location-primary-comment">
+        <% if @location.suppress_display %>
+          <p class="warn">Public display of this location is suppressed.</p>
```
Comment (elohanlon) [inline app/views/locations/show.html.erb]: Did you intend to use the `locales/en.yml` file for this message?

**Review by elohanlon — APPROVED** [review]
Looks good!  Just left one small question/suggestion in a comment, but it's so small that I'll approve now anyway.

File: app/helpers/edit_location_helper.rb
Comment (er-k) [inline app/helpers/edit_location_helper.rb:1]: i recommend bootstrapification of these checkboxes for readability to use `.form-check`
for example:
```
<li class="form-check">
<input class="form-check-input" type="checkbox" value="155" name="location[access_points][]" id="location_access_points_2" >
<label class="form-check-label" for="location_access_points_2" >John Jay Dining Hall 2024-07-12 10:52:14 -0400</label>
</li>
```
ref: https://getbootstrap.com/docs/4.6/components/forms/#default-stacked

**Review by er-k — APPROVED** [review]
other than the one minor form checkbox suggestion (which is on the admin side and not critical) that can be pushed to a future release, i approve!

File: app/helpers/edit_location_helper.rb
Comment (barmintor) [reply app/helpers/edit_location_helper.rb:1]: Done!

File: app/views/locations/show.html.erb (hunk 2 above)
Comment (barmintor) [reply app/views/locations/show.html.erb]: Yes!

**Pull request merged**